app/views/coronavirus_landing_page/components/shared/_timeline.html.erb

```diff
@@ -5,72 +5,54 @@
     margin_bottom: 2
   } %>
 
-  <% if details.show_timeline_nations? %>
-    <%= form_tag coronavirus_landing_page_path,
-      method: :get,
-      class: "js-change-location" do |f| %>
+  <%= form_tag coronavirus_landing_page_path,
+    method: :get,
+    class: "js-change-location" do |f| %>
 
-      <%= render "govuk_publishing_components/components/radio", {
-        name: "nation",
-        heading: t("coronavirus_landing_page.show.timeline.radio.heading"),
-        margin_bottom: 7,
-        items: details.timeline_nations_items
+    <%= render "govuk_publishing_components/components/radio", {
```

Comment:
I think this code might be missing, I haven't run it on integration so not 100% sure but I think the title and nesting the details component are still needed?

```
<p class="govuk-body">
        <%=t("coronavirus_landing_page.show.timeline.controls.status_html", nation: details.selected_nation.titleize) %>
      </p>

      <%= render "govuk_publishing_components/components/details", {
        title: t("coronavirus_landing_page.show.timeline.controls.change"),
        data_attributes: {
          track_category: "pageElementInteraction",
          track_action: "TimelineNation",
          module: "govuk-details",
        }
      } do %>
```

Reply:
Ah, the details element was removed in a later PR: https://github.com/alphagov/collections/pull/2465

Reply:
👍